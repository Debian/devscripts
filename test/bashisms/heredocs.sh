#!/bin/sh

cat <<- FOO
    foo
	bar
    moo
FOO

echo -e moo # BASHISM

foo() {
	cat <<- FOO
	foo
	bar
	moo
	FOO
	echo -e BASHISM
}

bar() {
	cat <<- FOO
	foo
	bar
	moo
    FOO
    echo -e nothing wrong here
FOO
	echo -e BASHISM
}


moo() {
	cat << FOO
	foo
	bar
	moo
    FOO
    echo -e nothing wrong here
	FOO
	echo -e still nothing wrong here
FOO
	echo -e BASHISM
}

baz() {
    cat << EOF1
EOF1 
    echo -e still inside the here doc
EOF1 ; echo -e still inside...
EOF1
    echo -e BASHISM
}
