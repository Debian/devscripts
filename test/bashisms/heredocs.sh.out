possible bashism in bashisms/heredocs.sh line 9 (echo -e):
echo -e moo # BASHISM
possible bashism in bashisms/heredocs.sh line 17 (echo -e):
	echo -e BASHISM
possible bashism in bashisms/heredocs.sh line 28 (echo -e):
	echo -e BASHISM
possible bashism in bashisms/heredocs.sh line 42 (echo -e):
	echo -e BASHISM
possible bashism in bashisms/heredocs.sh line 51 (echo -e):
    echo -e BASHISM
